workers/unity/Packages/com.improbable.gdk.core/Config/LocatorConfig.cs

```diff
@@ -16,10 +19,30 @@ public LocatorConfig()
         public override void Validate()
         {
             ValidateConfig(LocatorHost, RuntimeConfigNames.LocatorHost);
-            if (LocatorParameters.CredentialsType == LocatorCredentialsType.LoginToken)
+            switch (LocatorParameters.CredentialsType)
             {
-                ValidateConfig(LocatorParameters.LoginToken.Token, RuntimeConfigNames.LoginToken);
-                ValidateConfig(LocatorParameters.ProjectName, RuntimeConfigNames.ProjectName);
+                case LocatorCredentialsType.LoginToken:
+                    ValidateConfig(LocatorParameters.LoginToken.Token, RuntimeConfigNames.LoginToken);
+                    ValidateConfig(LocatorParameters.ProjectName, RuntimeConfigNames.ProjectName);
+                    break;
+                case LocatorCredentialsType.Steam:
+                    ValidateConfig(LocatorParameters.Steam.Ticket, RuntimeConfigNames.SteamToken);
+                    ValidateConfig(LocatorParameters.Steam.DeploymentTag, RuntimeConfigNames.SteamDeploymentTag);
+                    ValidateSteamDeploymentTag(LocatorParameters.Steam.DeploymentTag);
+                    break;
+            }
+        }
+
+        private void ValidateSteamDeploymentTag(string deploymentTag)
+        {
+            const string regex = @"^[A-Za-z0-9][A-Za-z0-9_]*$";
+            var match = Regex.Match(deploymentTag, regex);
```

Comment:
I think that it actually needs to be one specific tag:  `steam_deployment`
https://docs.improbable.io/reference/12.0/unitysdk/customize/steam#distributing-with-steam

Reply:
Here's where I derived the regex from: https://github.com/improbable/everything/blob/3da51c9907e18f0b52172cdb6dfb2c6bef04ce7e/worker_sdk/csharp/sdk/worker/Parameters.cs#L212

We can also hardcode the deployment tag to be "steam_deployment" if that's the case?